Dockerfile

```diff
@@ -9,17 +10,16 @@ RUN go mod download
 RUN go mod verify
 RUN CGO_ENABLED=0 go build -o sendsync
 
-ENV GITHUB_CLI_VERSION=1.14.0
-
 FROM alpine:latest as gh
-
+ARG GITHUB_CLI_VERSION
```

Comment:
con ARG funciono??
que raro que no haya funcionado con ENV... :(

Reply:
El tema era que en cada FROM se reseteaban las variables, por eso no lo tomaba en cuenta. Lo puse como ARG porque en verdad no se necesita pasar hacia el container y solo se usa para el build.

> An ARG declared before a FROM is outside of a build stage, so it can’t be used in any instruction after a FROM. To use the default value of an ARG declared before the first FROM use an ARG instruction without a value inside of a build stage.

https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact

Reply:
ya cache!!

Reply:
buena, si.. lo estaba poniendo en el primer from,,, y queriendo usarlo en el segundo